uaaextras/templates/email/body.html

```diff
@@ -11,31 +11,24 @@
             <a href="https://cloud.gov">{{branding.company_name}}</a>.
         </p>
         <p>
-            {{branding.company_name}} is a service by 18F intended to help
-            federal teams create and deliver quality digital services securely hosted in the cloud.
-            {{branding.company_name}} is currently in
-            <a href="https://18f.gsa.gov/dashboard/">alpha state</a>
-            and under active development. We'll ask for your feedback and make
-            improvements frequently.
+            {{branding.company_name}} is a service by 18F intended to help federal teams create and deliver quality digital services securely hosted in the cloud.
         </p>
         <strong>Your next steps</strong>
         <p>
         <ol>
             <li>
-                To log into cloud.gov, which requires multi-factor authentication, first install a mobile application that generates time-based one-time passwords.  We recommend installing either <a href="https://support.google.com/accounts/answer/1066447?hl=en">Google Authenticator</a> or <a href="https://www.authy.com/app/mobile">Authy</a> on your mobile device. In your next step, the cloud.gov registration process will give you instructions for using it.
+	    <b>Download a Multi-Factor Mobile Application</b> - cloud.gov requires multi-factor authentication, which you can download on your mobile device. We recommend either <a href="https://support.google.com/accounts/answer/1066447?hl=en">Google Authenticator</a> or <a href="https://www.authy.com/app/mobile">Authy</a>.
```

Comment:
Suggest "such as either" instead of "recommend either" to be more neutral about endorsing things.